src/components/author/author.scss

```diff
@@ -0,0 +1,9 @@
+@use "../../compiled/tokens/scss/color";
+
+/**
+ * Container
+ */
+
+.c-author {
+  color: color.$base-gray-dark;
```

Comment:
Seems to break with dark theme applied

<img width="502" alt="Screen Shot 2021-06-01 at 9 27 12 AM" src="https://user-images.githubusercontent.com/69633/120358644-920e0f80-c2bb-11eb-9327-eb1a9232b662.png">